src/app/components/apps-view/apps-view.component.scss

```diff
@@ -45,8 +44,9 @@
   .mat-cell.indicator-icon {
     max-width: 40px;
     font-size: 18px;
+    margin-left: 10px;
   }
-  .job-details {
+  .job-allocations {
```

Comment:
I thought all job related tags had been changed to apps in YUNIKORN-65 (#31) ?

Reply:
done